### ISSUE TITLE
Optical Character Recognition (OCR) functionality

### ISSUE THREAD

**Issue by AdhocAdam** [AdhocAdam/smletsexchangeconnector#5]
Some kind of OCR support that allows images to be parsed into text and then be appended to the Work Item Description or other field of one's choosing.

**Comment by AdhocAdam** [comment]
Through Azure Cognitive Services it's possible to perform text extraction (OCR) from images. Take the following PowerShell function with a deployed Azure Cognitive Service Vision API. Passing in your own region, api key, and image url it's possible to perform OCR against an image. Per Microsoft documentation a binary stream can also be used which would ultimately be used within the connector.


```powershell
$azureRegion = ""
$azureCogSvcVisionAPIKey = ""

function Get-AzureEmailImageText ($imageURLToEvalute)
{
        #define cognitive services URLs
        $imageTextURI = "https://$azureRegion.api.cognitive.microsoft.com/vision/v1.0/ocr?detectOrientation=true"

        #create the JSON request
        $messagePayload = $(@{"url"=$imageToEvalute} | ConvertTo-Json)
    
        #invoke the Cognitive Services Sentiment API
        $ocrResult = Invoke-RestMethod -Method Post -Uri $imageTextURI -Header @{ "Ocp-Apim-Subscription-Key" = $azureCogSvcVisionAPIKey } -Body $messagePayload -ContentType "application/json"
    
        #words = $ocrResult.regions.lines.words
        return $ocrResult
}
```

**Comment by AdhocAdam** [comment]
Will be released with #197 and #198
